MergerService/appsettings.Development.json

```diff
@@ -4,7 +4,9 @@
     "inputPath": "input",
     "gpkgPath": "gpkg",
     "filePath": "tiles",
-    "batchSize": 1000
+    "limitBatchSize": true,
+    "batchSize": 1000,
```

Comment:
Add sub-section in "GENERAL" called batchSize with those values

Reply:
done